lib/renaissance/auctions/auctions.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Renaissance.Auctions do
+  alias Renaissance.{Auction, Repo, Users}
+
+  def create_auction(email, params) do
+    auction =
+      params["auction"]
+      |> add_user_id(email)
+      |> format_end_date()
+      |> format_price()
+
+    Repo.insert(Auction.changeset(%Auction{}, auction))
+  end
+
+  def get_by_title(title) do
+    Repo.get_by(Auction, title: title)
+  end
+
+  defp format_end_date(params) do
+    datetime =
+      "#{params["end_date_day"]}T#{params["end_date_time"]}:00-05:00"
```

Comment:
is this just formatting it in CST because this looks very fragile

Reply:
Er, CDT, since -5, I think

Reply:
Resolved with 159a775e493da7136d042a4f76b690b4d8cb2aea